### PR TITLE
bump version to 1.1.1-beta

### PR DESCRIPTION
**Pull request by jgravois**

### CODE REVIEW

**Comment by jgravois** [comment]
noticed that .xml config files still said `1.0`

File: DotNet/Web.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version="1.1.1beta"?>
```
Comment (bsvensson) [inline DotNet/Web.config]: This is referring to the [version of XML](https://msdn.microsoft.com/en-us/library/ms256048(v=vs.110).aspx) standard that this file conforms to, not the version of the proxy.  I think we should stick with 1.0 here.

Comment (jgravois) [inline DotNet/Web.config]: thanks for clarifying.  fixed (and squashed).   this should be ready to merge.

File: DotNet/proxy.xsd [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.1.1beta" encoding="utf-8"?>
```
Comment (bsvensson) [inline DotNet/proxy.xsd]: @jgravois - I still see `xml version="1.1.1beta"` instead of `xml version="1.0"`.

**Comment by bsvensson** [comment]
@jgravois - I noticed you use the dashed string for the comment `1.1.1-beta` but a non-dashed for the string variable `1.1.1beta`. Is that on purpose? I looked at http://semver.org/ which seems to favor the dashed version, but maybe I missed something.

**Comment by jgravois** [comment]
no.  you're right.  fixed (and squashed).

**Comment by bsvensson** [comment]
Looks nice and clean - merged :)

**Pull request merged**